microservices-schema.json

```diff
@@ -0,0 +1,120 @@
+                ]
+            },
+            "spec": {
+                "type": "string",
+                "$id": "https://raw.githubusercontent.com/hmcts/reform-api-docs/master/docs/microservices.json/items/properties/spec",
+                "title": "API specs",
+                "description": "Link to API specification json file",
+                "default": null,
+                "examples": [
+                    "https://hmcts.github.io/reform-api-docs/specs/payments.json"
+                ]
+            },
+            "colour": {
+                "type": "string",
+                "$id": "https://raw.githubusercontent.com/hmcts/reform-api-docs/master/docs/microservices.json/items/properties/colour",
+                "title": "Network colout",
+                "description": "Colour used to paint and group APIs in the network graph",
+                "default": "",
+                "examples": [
+                    "#8e44ad"
+                ]
+            },
+            "dependencies": {
+                "type": "array",
+                "$id": "https://raw.githubusercontent.com/hmcts/reform-api-docs/master/docs/microservices.json/items/properties/dependencies",
+                "items": {
+                    "type": "object",
+                    "$id": "https://raw.githubusercontent.com/hmcts/reform-api-docs/master/docs/microservices.json/items/properties/dependencies/items",
+                    "properties": {
+                        "name": {
+                            "type": "string",
+                            "$id": "https://raw.githubusercontent.com/hmcts/reform-api-docs/master/docs/microservices.json/items/properties/dependencies/items/properties/name",
+                            "title": "API name",
+                            "description": "The name of the API, usually a match of GitHub repo README title",
+                            "default": "",
+                            "examples": [
+                                "IdAM"
+                            ]
+                        },
+                        "hard": {
+                            "type": "boolean",
+                            "$id": "https://raw.githubusercontent.com/hmcts/reform-api-docs/master/docs/microservices.json/items/properties/dependencies/items/properties/hard",
+                            "title": "???",
+                            "description": "???",
+                            "default": false,
+                            "examples": [
+                                true
+                            ]
+                        },
+                        "apis": {
+                            "type": "array",
+                            "$id": "https://raw.githubusercontent.com/hmcts/reform-api-docs/master/docs/microservices.json/items/properties/dependencies/items/properties/apis",
+                            "items": {
+                                "$id": "grizzlybear",
```

Comment:
grizzlybear?

Reply:
Schema generator populates when there is no information about the object (array items in this case)